packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -378,6 +378,10 @@ export default defineComponent({
 
     passwortProtectionTooltip() {
       return this.$gettext('This link is password-protected')
+    },
+
+    isAliasLink() {
+      return this.link.permissions === '0'
```

Comment:
No magic numbers please :-) Please have a look at `_buildLink` in `web-app-files/src/helpers/resources.ts`. That already loads the correct role object. You could add the role to the json that the function returns and utilize it here.